aospy/internal_names.py

```diff
@@ -42,8 +42,8 @@
 AVERAGE_T2_STR = 'average_T2'
 TIME_WEIGHTS_STR = 'time_weights'
 NV_STR = 'nv'
-AVG_START_DATE_STR = 'avg_start_date'
-AVG_END_DATE_STR = 'avg_end_date'
+AVG_START_DATE_STR = 'data_start_date'
+AVG_END_DATE_STR = 'data_end_date'
```

Comment:
Good catch.  I agree that `avg` is bad.  What about just `start_date` and `end_date`?

There's a bigger issue though: we should really be encoding into the datasets the start and end dates of the actual calculation, not the span of data from which it was loaded.

How difficult do you think that would be?  If not bad, you can include here.  Otherwise let's create a new issue for it.